### ISSUE TITLE
Change delimiter to include metacharacters, exclusing quotes

### ISSUE THREAD

**Issue by dennorak** [Eagle-9/crash#16]
```
A word is a sequence of characters considered as a single unit by the program. Words are separated by a
metacharacter if unquoted. The metacharacters includes
| & ; ( ) < > space tab newline
```

**Comment by dennorak** [comment]
Use enums for the classification

**Comment by Derp88** [comment]
Should be implemented with commit #ab6724f
As usual, extra testing is good if anyone wants to.

Note, I did not include tab as all tabs are converted to spaces by the whitespace checker.
